src/System.Data.SqlClient/tests/ExceptionTest.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System.Collections;
+using Xunit;
+
+namespace System.Data.SqlClient.Tests
+{
+    public class ExceptionTest
+    {
+        // test connection string
+        string conn_string = "server=tcp:server,1432;database=test;uid=admin;pwd=SQLDB;connect timeout=60;";
+
+        // data value and server consts
+        private const string badServer = "NotAServer";
+        private const string sqlsvrBadConn = "A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)";
+        private const string execReaderFailedMessage = "ExecuteReader requires an open and available Connection. The connection's current state is closed.";
+        private const string orderIdQuery = "select orderid from orders where orderid < 10250";
```

Comment:
It is preferable not to share parameters outside the tests function itself.
Especially if this query / Message is not used repeatably.

Because it will be much easier to see these message / query in the function itself, rather then have to lookup a shared variable somewhere on the top.

Especially, if the tests getting bigger, it may break the other tests if you change one of the shared variables. or this shared variables will accumulate and start hard to read which tests are using this variables.